CMSgov/onemac
# Oy2 21321

Story: https://qmacbis.atlassian.net/browse/OY2-21321
Endpoint: See github-actions bot comment

### Details

Add messaging for confirmation dialogs on RAI Response form upon submit.

### Changes

- Updated confirmation dialog to allow a JSX.Element body
- restructured OneMacFormConfig to have a declared type for confirmSubmit
- Added default messaging for RAI Response to form lib
- Used defaultConfirmSubmitRAI type to configure for RAIs for waivers (including appk) and medicaid spa
- Configured chip spa rai to override default message with chip specific RAI message

### Test Plan

1. Login as statesubmitter
2. Find a [medicaid spa, chip spa, initial waiver, renewal waiver, amendment waiver, app k waiver] in RAI Issued status
3. Invoke the Respond to RAI action either from dashboard or details page
4. Fill out required fields (attachments)
5. Click Submit
6. Ensure confirmation modal appears with specified text based on package type
7. Repeat for each type in step 2
